extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -797,18 +768,18 @@ private static void writeKeyValuePartitionedTableImpl(@NotNull final Partitioned
             while (constituentIterator.hasNext()) {
                 final ObjectVector<? extends Table> constituentVector = constituentIterator.next();
                 final List<String> partitionStrings = partitionStringsList.get(row);
-                final File relativePath = new File(destinationRoot, String.join("/", partitionStrings));
+                final File relativePath = new File(destinationRoot, String.join(File.separator, partitionStrings));
```

Comment:
This can be an issue because the relative path written down to metadata file would have delimiter of windows and Linux won't understand it.
I would need to test it though.
If its an issue, we would need to normalize the relative paths first at the time of reading to use the correct file separator.